docs/arch_apis.md

```diff
@@ -1,13 +1,10 @@
 # APIs
 
-## Skill API
+There are several API's used by OVOS including for weather inquiries, Wolfram-alpha and others
 
-The Skill API uses the Message Bus to communicate between Skills and wraps the interaction in simple Python objects making them easy to use.
+OVOS provides default keys that are used by default and no more configuration is needed.
 
-### Limitations
+You can always provide your own API keys for each service, and even add more APIs for OVOS to use.
```

Comment:
Suggest adding some examples. I know this is for less technical users, but to my mind, that means we should provide as detailed examples for configuration as possible.

Reply:
After we get a simple version out, I thought better examples could be added pretty easy then.  And I would like to keep most of the technical stuff in the other docs.  Make this one super simple